src/neuroconv/datainterfaces/behavior/sleap/sleapdatainterface.py

```diff
@@ -0,0 +1,70 @@
+"""Author: Heberto Mayorquin."""
+from typing import Optional
+from pathlib import Path
+
+from pynwb.file import NWBFile
+
+from ....basedatainterface import BaseDataInterface
+from ....tools.nwb_helpers import make_or_load_nwbfile
+from ....tools import get_package
+from ....utils import dict_deep_update, FilePathType, OptionalFilePathType
+
+
+class SleapInterface(BaseDataInterface):
```

Comment:
I know it's kind of minor, but references on their site are as SLEAP since it's an abbreviation. Should this then reflect that as the `SLEAPInterface`, like we do for other things like EDF, CED, SpikeGLX, etc?

Reply:
That makes sense to me. Let's see what @bendichter decides.

Reply:
I vote for SLEAP

Reply:
All right .I changed it here and on the gallery.